_posts/molecules/2018-1-2-breadcrumbs.html

```diff
@@ -7,7 +7,9 @@
 
 %nav{"aria-label" => "breadcrumb"}
   %ol.breadcrumb
-    %li.breadcrumb-item.active{"aria-current" => "page"} Home
+    %li.breadcrumb-anchor
+      = link_to('#', title: 'Open Build Service') do
```

Comment:
This doesn't work since it's Rails helper. You can leave it here, but as a comment and then add the `%a`.